packages/frontend/src/components/common/modal/ChartCreateModal.tsx

```diff
@@ -118,14 +126,14 @@ const ChartCreateModal: FC<ChartCreateModalProps> = ({
     );
 
     const handleSaveChartInDashboard = useCallback(
-        async (values) => {
+        async (values: ChartCreateModalFormValues) => {
             if (!fromDashboard || !unsavedDashboardTiles || !dashboardUuid)
                 return;
             const newChartInDashboard: CreateChartInDashboard = {
                 ...savedData,
                 name: values.name,
                 description: values.description,
-                dashboardUuid: values.dashboardUuid,
+                dashboardUuid: dashboardUuid,
```

Comment:
good spot, that's so easy to dismiss. Guess rule of thumb is to always pass a generic to the `useForm` from now on 👍

Reply:
Yea, I thought it would do better with the types. 

I want to add a test too.